### ISSUE TITLE
latest `jupyterlab-widgets` appears to break things

### ISSUE THREAD

**Issue by kushalkolar** [vispy/jupyter_rfb#58]
It seems like the latest `jupyterlab-widgets` released in the past month breaks the working of the remote frame buffer. Running the `hello_world` example with `jupyterlab-widgets` v3.0.3 (haven't tested with 3.0.0 - 3.0.2) produces the following when you try to create the output in a jupyter cell.

This issue is not present when using the previous release of `jupyterlab-widgets`, i.e. v1.1.1, and `ipywidgets` v7.7.0.

```javascript
[Open Browser Console for more detailed log - Double click to close this message]
Model class 'RemoteFrameBufferModel' from module 'jupyter_rfb' is loaded but can not be instantiated
TypeError: r._deserialize_state is not a function
    at f._make_model (http://localhost:8891/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.3e1e5adfd821b9b96340.js?v=3e1e5adfd821b9b96340:1:7933)
```

![image](https://user-images.githubusercontent.com/9403332/189465378-ddd96f04-3c09-4d31-9842-93e3b91c019c.png)

**Comment by djhoese** [comment]
Given the version number jump to 3.0 and ipywidgets jumping to 8.0 with major custom widget changes, my guess is that jupyter_rfb is not compatible with these new versions and will require a decent amount of work to get updated. Looking at

https://ipywidgets.readthedocs.io/en/stable/migration_guides.html

It seems the suggestion is to regenerate the entire python package structure from the cookiecutter project. This shouldn't be too big of a deal, but potentially having to rename properties/traits to work with this new version is going to take some time. The lead developer of this project is currently on paternity leave and likely won't have time to update this.

@kushalkolar Do you or someone you know have time to look into updating the library to work with the new versions of the widgets libraries? I think we can focus on dropping support for the older versions if necessary to keep the code cleaner.

**Comment by kushalkolar** [comment]
I could try this out, but I can't provide a timeline. Meanwhile, should `jupyter_rfb` make a bugfix release? Probably sufficient to change `install_requires` in `setup.py`, I can make a PR.

**Comment by djhoese** [comment]
That sounds good to me. I'll have to update the conda-forge recipe too after that release, but that's not too bad.

**Comment by almarklein** [comment]
Closed by #64
